scaleway/resource_container_namespace.go

```diff
@@ -0,0 +1,178 @@
+package scaleway
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	container "github.com/scaleway/scaleway-sdk-go/api/container/v1beta1"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+func resourceScalewayContainerNamespace() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceScalewayContainerNamespaceCreate,
+		ReadContext:   resourceScalewayContainerNamespaceRead,
+		UpdateContext: resourceScalewayContainerNamespaceUpdate,
+		DeleteContext: resourceScalewayContainerNamespaceDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Default: schema.DefaultTimeout(defaultContainerTimeout),
```

Comment:
Should be defaultContainerNamespaceTimeout other resource related to container will have timeout

Reply:
Constant updated